server/http/src/server.rs

```diff
@@ -188,9 +188,9 @@ mod tests {
                     schema_sink.send(input_event).wait().unwrap();
 
                     // Wait for the schema to be received and extract it.
-                    // Wait for thirty seconds for that to happen, otherwise fail the test.
+                    // Wait a while for that to happen, otherwise fail the test.
                     let start_time = Instant::now();
-                    let max_wait = Duration::from_secs(30);
+                    let max_wait = Duration::from_secs(60);
```

Comment:
Huh why does this take so long? If it's taking that long, how about thread sleep 1ms instead of yield_now?

Reply:
It was just travis being slow, it's usually fast.

Reply:
I'm wondering if sleeping could speed that up

Reply:
ah got it, done